compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -26,6 +26,47 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None):
 
   var myClassLoader: AbstractFileClassLoader = uninitialized
 
+  private var pprintState: ((Any, Int, Int, Int) => String) | Null = null
+  private def pprintRender(value: Any, width: Int, height: Int, initialOffset: Int)(using Context): String = {
+    def fallback = (value: Any, width: Int, height: Int, initialOffset: Int) =>
+      dotty.shaded.pprint.PPrinter.BlackWhite.apply(
+        value, width = 100, height = 50, initialOffset = initialOffset
```

Comment:
Perhaps we should put width/height numbers into constants, since we're also repeating it down below

Reply:
good catch, I now passed the parameters along explicitly.